applications/hpf/gpio/README.rst

```diff
@@ -10,7 +10,7 @@ High-Performance Framework GPIO
 
 .. caution::
 
-   The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 device.
+   The High-Performance Framework (HPF) support in the |NCS| is :ref:`experimental <software_maturity>` and is limited to the nRF54L15 and nRF7120 devices.
```

Comment:
@bama-nordic @ioannisg Is it fine to mention **nRF7120 devices** in the documentation currently?

Reply:
Agree, as per offline discussion with him, it's fine to mention this in the docs but not in Changelog and Release notes.